src/test/resources/fixtures/codeStyle/parameters-default-equals.fixed.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+function method($a = 0, $b = 0) {
```

Comment:
* Replaced "method()" to "test()";
* Now testing method calls too (now I found 51 cases on my project!);

Done (pushing now).